packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -265,6 +279,8 @@ class ContainerView: RCTView {
         playIfNeeded()
     }
     
+    
```

Comment:
I think it might be time to install SwiftLint for this Project. No reason for these things to show up every time someone accidentally creates a new empty line